### ISSUE TITLE
7-ZipPortable_16.04.paf.exe sourceforge address changed

### ISSUE THREAD

**Issue by AlexECX** [MarioMasta64/EverythingPortable#5]
EDIT:Well, it seems changing the address still results in a broken 7-zip intaller. Just go to the new address on sourceforge to download it directly and drop it in the /bin directory.

After looking through the code and trying a couple lines by themselves, I finaly found that launch_steam.bat (and the other launchers i presume) wasn't working because the downloaded 7-ZipPortable_16.04.paf.exe was bogus, or "broken" as windows says it. Turns out sourceforge changed the address from **https://sourceforge.net/portableapps/7-Zip%20Portable/** to **https://sourceforge.net/projects/portableapps/files/7-Zip%20Portable/**. 

For anyone who wants make it work, just open the launcher .bat with notepad, look (or ctrl+F) for the web address and replace it. Modifying the address in the .bat solved the issue.

Just posting this here if someone still wants to use SteamPortable (It still seems to work to date) and other portable launchers.

**Comment by MarioMasta64** [comment]
uh.. the url still works fine yknow?
![image](https://user-images.githubusercontent.com/20806802/37182935-949698ce-22e8-11e8-92b7-38ef55f5e72a.png)
![image](https://user-images.githubusercontent.com/20806802/37182941-9ae71e9c-22e8-11e8-9264-dc89bfd150f1.png)

**Comment by MarioMasta64** [comment]
you cannot use this url in a browser no, because it is a direct link, like with github there is a browsable link and the download directory, please use the launcher before reporting an issue. or if you have provide proof it doesnt work and more details so that i may fix it.

**Comment by AlexECX** [comment]
Well I tried the launcher again and it worked, but I also tried the url (the original one) and it properly sent me to the download page this time. I guess it just so happen that when I decided to try everything portable the page was down (the site did mention something about temporary unavailability *at that time).

Edit: Also, how the problem appeared at first was an infinite looping of the cursor blinking up an down, and was displaying and erasing a message extremely fast. Now I can't replicate the issue because I didn't keep a copy of the broken 7-zip, but it was happening on line 83, in the 7zip installer check, where this command was executed: .\extra\7-ZipPortable_16.04.paf.exe /destination="%CD%\bin\".

Anyway, the issue is fixed with the sourceforge page being up again.
